### PR TITLE
Add WAN sync statistics to diagnostics

### PR DESCRIPTION
**Pull request by blazember**
EE PR: https://github.com/hazelcast/hazelcast-enterprise/pull/2475

### CODE REVIEW

**Comment by blazember** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalWanPublisherStatsImpl.java [hunk 1]
```diff
@@ -103,16 +105,24 @@ public void setSentCacheEventCounter(Map<String, DistributedObjectWanEventCounte
...
         return lastConsistencyCheckResults;
     }
 
+    public void setLastSyncStats(Map<String, WanSyncStats> lastSyncStats) {
+        this.lastSyncStats = lastSyncStats;
+    }
+
+    @Override
+    public Map<String, WanSyncStats> getLastSyncStats() {
+        return lastSyncStats;
```
Comment (mmedenjak) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalWanPublisherStatsImpl.java]: Hm, we could probably add the check and sync results to the JSON but we can do that in a separate PR.

File: hazelcast/src/main/java/com/hazelcast/monitor/LocalWanPublisherStats.java [hunk 2]
```diff
@@ -80,4 +81,9 @@
      * Returns the last results of the consistency checks, mapped by map name.
      */
     Map<String, ConsistencyCheckResult> getLastConsistencyCheckResults();
+
+    /**
+     * Returns the last synchronization statistics, mapped by map name.
+     */
+    Map<String, WanSyncStats> getLastSyncStats();
```
Comment (ahmetmircik) [inline hazelcast/src/main/java/com/hazelcast/monitor/LocalWanPublisherStats.java]: maybe we can enhance javaDoc to tell why only the last sync stats is important?

Comment (blazember) [reply hazelcast/src/main/java/com/hazelcast/monitor/LocalWanPublisherStats.java]: :+1: I'll add this in a separate PR, merging this one for now.
